packages/xo-web/src/xo-app/hub/resource.js

```diff
@@ -171,10 +180,72 @@ export default decorate([
       redirectToTaskPage() {
         this.props.router.push('/tasks')
       },
+      showDescription() {
+        const {
+          data: { public: _public },
+          name,
+        } = this.props
+        alert(
+          name,
+          <div>
+            {isEmpty(omit(_public, BANNED_FIELDS)) ? (
+              <p>{_('hubTemplateDescriptionNotAvailable')}</p>
+            ) : (
+              <div>
+                <Ul>
+                  {EXCLUSIVE_FIELDS.map(fieldKey => {
+                    const field = _public[fieldKey]
+                    return field !== undefined ? (
+                      <Li key={fieldKey}>
+                        <div
+                          dangerouslySetInnerHTML={{
+                            __html: marked(field),
```

Comment:
`EXCLUSIVE_FIELDS` doesn't necessarily contain "markdownable" properties so I think you should specifically test for `longDescription` here.